packages/tracing/src/browser/web-vitals/README.md

```diff
@@ -0,0 +1,14 @@
+# web-vitals
+
+> A modular library for measuring the [Web Vitals](https://web.dev/vitals/) metrics on real users.
+
+This was vendored from: https://github.com/GoogleChrome/web-vitals
```

Comment:
@dashed could you please add the commit SHA to this README?

It makes it easier for us to see what's changed upstream and make updates later.

Reply:
I'll update it to https://github.com/GoogleChrome/web-vitals/tree/56c736b7c4e80f295bc8a98017671c95231fa225